spec.html

```diff
@@ -22476,7 +22476,7 @@ <h1>Static Semantics: Early Errors</h1>
       <emu-grammar>WithStatement : `with` `(` Expression `)` Statement</emu-grammar>
       <ul>
         <li>
-          It is a Syntax Error if the code that matches this production is contained in strict mode code.
+          It is a Syntax Error if the code matched by this production is contained in strict mode code.
```

Comment:
```suggestion
          It is a Syntax Error if the source text matched by this production is strict mode code.
```

Reply:
I made the "source text" change, but not the "contained in" deletion. While I believe it's correct, the reason for its correctness is a bit deeper than everything else in this PR.